.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish npm package
+
+on:
+  push:
+    branches:
+      - master
+      - github-package
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20'
```

Comment:
I'm afraid since we're using Meteor we're quite limited in the Node version we can use 😞 It's listed as `v14.19.1` in `.nvmrc`, so should we use the same here for consistency?

Reply:
As we are using `npm ci` instead of `npm i`, and we don't directly use the Node, we shouldn't have an issue.

Reply:
Yeeeah, you're probably right, but I'm not 100% sure that `npm run build` provides the same results regardless fo the node version. `npm run build` does use Node, so version difference might make a difference here.

Reply:
Updated (beta1):
https://github.com/CodeSignal/meteor-protomongo/actions/runs/9876015294/job/27274083938
https://github.com/CodeSignal/meteor-protomongo/pkgs/npm/meteor-protomongo/241851387

Reply:
Thank you!